LibreNMS/Alert/AlertRules.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace LibreNMS\Alert;
+
+use App\Models\Device;
+
+class AlertRules
+{
+    /**
+     * Check if device is under maintenance
+     * @param int $device_id Device-ID
+     * @return bool
+     */
+    public function IsMaintenance($device_id)
+    {
+        return \App\Models\Device::find($device_id)->isUnderMaintenance();
+    }
+
+    /**
+     * Run all rules for a device
+     * @param int $device_id Device-ID
+     * @return void
+     */
+    public static function CheckRules($device_id)
+    {
+
+        //Check to see if under maintenance
+        if (IsMaintenance($device_id) > 0) {
```

Comment:
This is failing tests: https://travis-ci.com/librenms/librenms/jobs/189340179

`$this->IsMaintenance()`?

Reply:
No worries, I've edited the title to WIP so we know. Remove it when you're ready